docs/docker/README.MD

```diff
@@ -1,5 +1,12 @@
 ![](docker-for-beginners.png)
-# Docker Images
+
+# What is Docker?
+Docker is a tool used by developers to package together dependencies into a single container (or image). What this means for *you* is that in order to use your integration, you are not required to "pip install" all of the packages required. They are part of a container that "docks" to the serverand contains all of the libraries you need. To learn more about docker, [visit their site here](https://docs.docker.com/)
+
+# Why Use Docker?
+Primarily we use docker to run python scripts in a controlled environment. They run isolated from the server to prevent someone from accidentally damaging the server. By packaging libraries and dependencies together, we can prevent unknown issues from occurring since the environment is all the same. Similarly, by using a container as opposed to installing the dependencies manually, we prevent accidentally installing malicious packages.
```

Comment:
remove this `Similarly, by using a container as opposed to installing the dependencies manually, we prevent accidentally installing malicious packages`
because we will not release docker image that has malicious package. We are running process that checks for package vulnerability before we release a docker